.github/workflows/check-pr-title.yml

```diff
@@ -11,5 +11,5 @@ jobs:
     steps:
       - uses: deepakputhraya/action-pr-title@master
         with:
-          regex: '^(feat|fix|build|chore|test|refactor|docs)(\(.+\))?!?: [A-Z].+$'
+          regex: '^(build|chore|docs|feat|fix|perf|refactor|test)(\(.+\))?!?: [A-Z].+$'
```

Comment:
I would opt for only allowing prefixes that we use or want to use; if we're not clear what `build` is supposed to indicate (and I'm not), we should either come up with a useful definition of it - or remove it entirely. In what cases would you find it useful?

Reply:
It could be useful for indicating changes in our continuous integration setup building the software. So essentially this PR could even use it as it is not really a maintenance tasks indicated by "chore" but rather an improvement to the way we are building and publishing the software.

Reply:
@seakayone build can be deleted. I added it long time ago, due to some people which was using `build` instead of `chore`. But now as we have the regex, this can be removed. At the both are displayed under the same label in release notes.

I see the specification, but there is nothing about `perf`. Does it stand for performance? Do we really need this? Wouldn't it be actually the same as `refactor`? Or at least very close to?

Reply:
The spec only requires `feat` and `fix`. I think all the others make sense.